include/seqan3/range/view/get.hpp

```diff
@@ -0,0 +1,100 @@
+ */
+
+#pragma once
+
+#include <seqan3/std/view/transform.hpp>
+
+namespace seqan3::view
+{
+/*!\name Alphabet related views
+ * \{
+ */
+
+/*!\brief               A view calling std::get on each element in a range.
+ * \tparam size_t       The type of the range being processed. See below for requirements. [template parameter is
+ *                      omitted in pipe notation]
+ * \tparam index        The index to get.
+ * \param[in] urange    The range being processed. [parameter is omitted in pipe notation]
+ * \returns             A range of std::get. See below for the properties of the returned range.
+ * \ingroup view
+ *
+ * ### View properties
+ *
+ * This view is a **deep view:** Given a range-of-tuples as input (as opposed to just a range), it will apply
+ * the transformation on the innermost tuple (instead of the outermost range).
+ *
+ * | range concepts and reference_t  | `urng_t` (underlying range type)      | `rrng_t` (returned range type)                     |
+ * |---------------------------------|:-------------------------------------:|:--------------------------------------------------:|
+ * | std::ranges::InputRange         | *required*                            | *preserved*                                        |
+ * | std::ranges::ForwardRange       |                                       | *preserved*                                        |
+ * | std::ranges::BidirectionalRange |                                       | *preserved*                                        |
+ * | std::ranges::RandomAccessRange  |                                       | *preserved*                                        |
+ * | std::ranges::ContiguousRange    |                                       | *lost*                                             |
+ * |                                 |                                       |                                                    |
+ * | std::ranges::ViewableRange      | *required*                            | *preserved*                                        |
+ * | std::ranges::View               |                                       | *preserved*                                        |
+ * | std::ranges::SizedRange         |                                       | *preserved*                                        |
+ * | std::ranges::CommonRange        |                                       | *preserved*                                        |
+ * | std::ranges::OutputRange        |                                       | *preserved*                                        |
+ * | seqan3::const_iterable_concept  |                                       | *preserved*                                        |
+ * |                                 |                                       |                                                    |
+ * | seqan3::reference_t             | std::tuple                            |                                                    |
```

Comment:
change this line to:
` * | seqan3::reference_t             | seqan3::tuple_like_concept            | `decltype(get<index>(seqan3::reference_t<urng_t>{}))` | `

Reply:
`std::tuple_element_t<index, seqan3::reference_t<urng_t>>` may be nicer for the second column